.pre-commit-config.yaml

```
@@ -21,7 +21,13 @@ repos:
         entry: prettier --write .
         files: \.(md|ya?ml)$
         language: node
-        additional_dependencies: ['prettier@3.3.3']
+        additional_dependencies: ["prettier@3.3.3"]
+  - repo: https://github.com/biomejs/pre-commit
+    rev: v0.4.0
```

```suggestion
    rev: v0.6.1
```

https://github.com/biomejs/pre-commit/releases/tag/v0.6.1